migrate/migrations/20191230125304-calendar-subscriptions.sql

```diff
@@ -0,0 +1,11 @@
+-- +migrate Up
+CREATE TABLE calendar_subscriptions (
+    id uuid NOT NULL UNIQUE,
+    name TEXT NOT NULL,
+    user_id uuid NOT NULL REFERENCES users(id),
+    last_access TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    disabled BOOLEAN NOT NULL DEFAULT false
+);
```

Comment:
We will need to store the `schedule_id` and `config` here as well:
https://github.com/target/goalert/compare/calendar-subscription-api#diff-04e3af9ffc876a383b6f192a8ff8450dR282